### ISSUE TITLE
Bootstrap indirect effects for mediation tests.

### ISSUE THREAD

**Issue by NicholasDanks** [ISS-Analytics/seminr#90]
- [x] Modify bootstrap_model() method to include indirect effects
- [x] ??? Modify bootstrap_model method to include total effects
- [ ] Cite Zhao et al (2010)
- [ ] Use Zhao et al (2010) to make conclusion about mediation?
- [ ] Do we need to bootstrap total effects?

**Comment by soumyaray** [comment]
Before we close this.. could we use Zhao's flowchart logic to also refer to what type of mediation is at hand?

**Comment by NicholasDanks** [comment]
Absolutely. And we need to have a good think about whether there is a need for bootstrapped TOTAL effects and all possible INDIRECT effects.

**Comment by soumyaray** [comment]
Why are we modifying bootstrap_model?

**Comment by JulianGaviriaL** [comment]
For the sake of completeness, it is informative to obtain total effects as well.

Best regards,
